### PR TITLE
#28 Add license types

### PR DESCRIPTION
**Pull request by hilton**

### CODE REVIEW

File: docs/licenses.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. _license_types:
+
+License types
+=============
+
+To log in to Signavio Workflow, your user account have a user license that has not expired.
```
Comment (SilentGert) [inline docs/licenses.rst]: Is there a word missing in the sentence like _must_?

File: docs/licenses.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+To downgrade a license to a *Developer Cloud* license, use the `purchasing <https://workflow.signavio.com/buy>`_ page.
+
+
+Academic Cloud
+--------------
+
+A free *Academic Cloud* license allows university students to use Signavio Workflow to learn about workflow management.
+You may only use this license if you have a valid university email address.
+You may only use this license for educational purposes and not for administrative processes in an educational institution.
+To register for an academic license, use the *academic version* link at the bottom of the `purchasing <https://workflow.signavio.com/buy>`_ page.
```
Comment (SilentGert) [inline docs/licenses.rst]: Should you mention that for now they have to register for a normal trial and can upgrade to the academic version afterwards?

**Comment by SilentGert** [comment]
@hilton Could you have a look at the comments?

File: docs/licenses.rst (hunk 1 above)
Comment (hilton) [inline docs/licenses.rst]: Yes. I’ve updated the pull request.

File: docs/licenses.rst (hunk 2 above)
Comment (hilton) [inline docs/licenses.rst]: I’ve updated the pull request with a commit for this that adds a sentence for both the _Developer Cloud_ and _Academic Cloud_ sections.

**Comment by SilentGert** [comment]
Looks good.

**Pull request merged**